### PR TITLE
Upgrade to Akka 2.4.6 (closes #10)

### LINKED ISSUE #10: Upgrade to Akka 2.4.6

### PR DESCRIPTION
**Pull request by hseeberger**

### CODE REVIEW

**Comment by patriknw** [comment]
there will unfortunately (thanks osgi) be a 2.4.6 very soon

**Comment by hseeberger** [comment]
When will you finally drop f...ing OSGi?

**Comment by patriknw** [comment]
like today, if it wasn't for the users that actually use it

File: build.sbt [hunk 1]
```diff
@@ -35,7 +35,7 @@ testOptions in Test += Tests.Argument("-oDF")
 // -a Show stack traces and exception class name for AssertionErrors.
 testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a")
 
-val AkkaVersion = "2.4.4"
+val AkkaVersion = "2.4.5"
```
Comment (patriknw) [inline build.sbt]: please change to 2.4.6

**Comment by hseeberger** [comment]
@patriknw, updated to 2.4.6.

**Comment by patriknw** [comment]
thanks

**Pull request merged**